ui/src/utils/validation-utils.ts

```diff
@@ -35,3 +35,37 @@ export function isPropEditFieldCommitted<T, TKey extends keyof T>(
 
 export const validate = <T>(isValid: boolean, issue: FieldValidationIssue<T>) =>
     isValid ? undefined : issue;
+
+export function getVisibleErrorsByProp<T>(
+    committedFields: (keyof T)[],
+    validationIssues: FieldValidationIssue<T>[],
+    prop: keyof T,
+): string[] {
+    return committedFields.includes(prop)
+        ? validationIssues.filter((error) => error.field === prop).map((error) => error.reason)
+        : [];
+}
```

Comment:
Meidän olemassaolevat virhehallinnan helpperit voisivat kaivata hieman rakkautta (tämä esim. palauttaa myös warningit nimestään huolimatta.) Lisäksi meillä saattaa olla paikkoja missä tällaiset about samanlaiset funktiot on duplikoitu, ja ne voisi yhdistää käyttämään näitä uusia, yhteisiä funktioita. Tällä reviewillä on kuitenkin tarpeeksi mittaa jo nyt, joten en lähtenyt refaktoroimaan niitä vielä tässä.